### DISCUSSION TITLE
HTTP to HTTPS redirect from 8080 to 443

### DISCUSSION THREAD

**Discussion by nfasoli** [globaleaks/globaleaks-whistleblowing-software#3834] [General]
Hi everybody
if i try to access my gl via 8433 everything is ok
But if a try to access via 8080 (i need to, i have to put the container behind an azure app gateway) it redirects me to 443

`
root@git:/home/utente/Desktop/GlobaLeaks# wget 172.17.0.2:8080
--2023-12-02 11:56:59--  http://172.17.0.2:8080/
Connecting to 172.17.0.2:8080... connected.
HTTP request sent, awaiting response... 301 Moved Permanently
Location: https://172.17.0.2/ [following]
--2023-12-02 11:56:59--  https://172.17.0.2/
Connecting to 172.17.0.2:443... failed: Connection refused.

`

any suggestion? I dont see any config out there

**Comment by evilaliv3** [comment]
Hello; this is correct.

For reasons of security globaleaks consider to be exposed publicly on port 80 and 443 and implements many security constraints around this assumption

**Comment by nfasoli** [comment]
Ok but i need to implement it on an Azure Container App behind a reverse proxy with TLS termination

I don't need SSL at all, so I need to disable it to publish as container (right now the app is exposed via vm). Container doesn't allow iptables mangling

Inviato da Outlook<http://aka.ms/weboutlook>
Da: Giovanni Pellerano ***@***.***>
Inviato: mercoledì 13 dicembre 2023 15:24
A: globaleaks/GlobaLeaks ***@***.***>
Cc: Nicola Fasoli ***@***.***>; Author ***@***.***>
Oggetto: Re: [globaleaks/GlobaLeaks] HTTP to HTTPS redirect from 8080 to 443 (Discussion #3834)


Hello; this is correct.

For reasons of security globaleaks consider to be exposed publicly on port 80 and 443 and implements many security constraints around this assumption

-
